src/release/issue-triaging.md

```diff
@@ -1,125 +1,274 @@
+The [`rust-lang/rust`] repository has thousands of issues and hundreds of people
+working on it. It is impossible for all people to check and solve issues. The
+goals of triaging are to:
 
-In practice, it is unrealistic for all issues to be solved quickly and found by right people.
-Through applications of labels we make the issue tracker more searchable for future reference,
-so that people in the future have an easier time finding related issues or issues they are interested
-in working on.
+- Improve **discoverability**:
+    - Categorize which teams an issue may concern.
+    - Make sure regressions are properly tagged and tracked.
+    - Connect issues to the relevant people (especially relevant teams).
+- First-pass **coarse classification** of the issue nature:
+    - Is this issue a bug, or is this discussion, or is this a user-side
+      problem?
+- Identify if the issue is **sufficiently actionable**:
+    - Does the report need further info, like reproduction steps, a reproducer,
+      or build/execution environment info?
+    - Does the reproducer need minimization?
+- Make it easier for contributors to **filter** issues by applying suitable
+  labels.
 
-Triaging can be done by **everyone**, no matter your permissions. We encourage everyone to help here,
-as triaging work is highly parallelizable and easy to get started with.
+In practice, it is unrealistic for all issues to be solved quickly and found by
+right people. Through applications of labels we make the issue tracker more
+searchable for future reference, so that people in the future have an easier
+time finding related issues or issues they are interested in working on.
+
+Triaging can be done by **everyone**, no matter your permissions. We encourage
+everyone to help here, as triaging work is highly parallelizable and easy to get
+started with.
 
 ## Initial triaging
 
-When an issue is opened, it gets the `needs-triage` label. This ensures that every issue gets an initial
-look and that no issue is ignored, or that when it is ignored, it is at least visibly ignored by still having the label.
+When an issue is opened, it usually receives the `needs-triage` label. This
+ensures that every issue gets an initial look and that no issue is never looked
+at.
 
-`needs-triage` is an initial checkpoint. The effort needed to get an issue past the label should be small.
+`needs-triage` is an initial checkpoint. The effort needed to get an issue past
+the label should be small.
 
-To do the initial triage and remove the `needs-triage` label, the following conditions should be fulfilled/considered.
-It's okay if not all of these are always considered; treat it as a guideline, not a hard checklist. It is also not exhaustive.
+To do the initial triage and remove the `needs-triage` label, the following
+conditions should be fulfilled/considered. It's okay if not all of these are
+always considered; treat it as a guideline, not a hard checklist. It is also not
+exhaustive.
```

Comment:
Can we remove reformatting changes from this patch or is it too inconvenient? I'd also probably stick to the current formatting choice for this document (which IIUC has no chars/line limit)

Reply:
I'll undo the reformatting, it was just more convenient when editing, and go with the no-column limit / sentence wrapping. We don't have any consistent formatting choices for docs in this repo, lol.

Reply:
Undid formatting changes, now all should be unbounded in column width.